build/metacontroller/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -eu
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+echo "generating Metacontroller resource definitions..."
+ytt -f "${SCRIPT_DIR}/_vendir/manifests" \
+    -f "${SCRIPT_DIR}/pin-image-to-digest.yml" > "${SCRIPT_DIR}/../../config/_ytt_lib/metacontroller/rendered.yml"
```

Comment:
@davewalter how about just using get-kbld.io to resolve digests. `kbld -f _vendir/manifests > ...`

Reply:
@cppforlife Hi Dmitriy - thanks for the input. We have created a [chore](https://www.pivotaltracker.com/story/show/172570472) to investigate usage of kbld in the future. For the purposes of this PR, we suggest leaving it as is for the following reasons:
1. It adds an additional dependency to the build chain that isn't currently used in this fashion.
2. There are other cases that might merit usage as well; We'd like to keep everything as consistent as possible.